transformer/upstream.js

```diff
@@ -22,29 +23,39 @@ const makeHMRConfig = require('babel-preset-react-native/configs/hmr');
 const path = require('path');
 const resolvePlugins = require('babel-preset-react-native/lib/resolvePlugins');
 
-const {compactMapping} = require('react-native/packager/src/Bundler/source-map');
+const {compactMapping} = require('./src/Bundler/source-map');
+
+import type {Plugins as BabelPlugins} from 'babel-core';
```

Comment:
Oooh, they are in the updated file. I was going off of the code base in master as-is.

Ok this should be easy. Something like `const BabelPlugins = require("babel-core").Plugins`